gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.6-bin.zip
```

Comment:
I see a change in `gradle-wrapper.properties`, but to get the new wrapper features (like checksum validation) it is required to regenerate wrapper jar (`./gradlew wrapper`).